src/extension/background-script/CryptoServices/utils.ts

```diff
@@ -48,6 +50,7 @@ export function makeTypedMessage(content: string, meta?: ReadonlyMap<string, unk
 
 interface KnownMetadata {
     [RedPacketMetaKey]: RedPacketJSONPayload
+    poll: PollMetaData
```

Comment:
Please rebase to the master, the `KnownMetadata` has been removed.

Reply:
ok